packages/helix-shared-config/src/IgnoreConfig.js

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright 2018 Adobe. All rights reserved.
+ * This file is licensed to you under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License. You may obtain a copy
+ * of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under
+ * the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+ * OF ANY KIND, either express or implied. See the License for the specific language
+ * governing permissions and limitations under the License.
+ */
+
+const BaseConfig = require('./BaseConfig.js');
+const GlobfileParser = require('./parsers/GlobfileParser.js');
+
+const IGNORE_CONFIG = '.hlxignore';
+
+class IgnoreConfig extends BaseConfig {
```

Comment:
I think this awkward, since all config is YAML based and has respective methods, like `toYAML` or `toJSON`.

Reply:
Mostly agree, @trieloff suggested this in https://github.com/adobe/helix-admin/issues/148
I don't necessarily see the need for it either, but I'm not adamantly opposed so I added it.

Reply:
all good. thanks for adding this here.